ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6108453?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6108453?buildTab=artifacts#/) on master @ [f4042d47fa8062a612c38d4696eb6bee9cee7c21](https://github.com/cockroachdb/cockroach/commits/f4042d47fa8062a612c38d4696eb6bee9cee7c21):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:162: test logs captured to: /artifacts/tmp/_tmp/045fecfe9e18c9da6f776a53d446f2c5/logTestTenantStreamingMultipleNodes1644022259
    test_log_scope.go:80: use -show-logs to present logs inline
    stream_replication_e2e_test.go:286: error executing '
          CREATE TABLE d.scattered (key INT PRIMARY KEY);
          INSERT INTO d.scattered (key) SELECT * FROM generate_series(1, 1000);
          ALTER TABLE d.scattered SPLIT AT (SELECT * FROM generate_series(20, 980, 20));
          ALTER TABLE d.scattered SCATTER;
          ': dial tcp 127.0.0.1:34645: connect: connection refused
    panic.go:500: -- test log scope end --
--- FAIL: TestTenantStreamingMultipleNodes (1191.45s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/tenant-streaming
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-18631

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6124518?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/6124518?buildTab=artifacts#/) on master @ [b173a16715e71e94115820374da1eb350b3b459d](https://github.com/cockroachdb/cockroach/commits/b173a16715e71e94115820374da1eb350b3b459d):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:162: test logs captured to: /artifacts/tmp/_tmp/045fecfe9e18c9da6f776a53d446f2c5/logTestTenantStreamingMultipleNodes156001989
    test_log_scope.go:80: use -show-logs to present logs inline
    stream_replication_e2e_test.go:286: error executing '
          CREATE TABLE d.scattered (key INT PRIMARY KEY);
          INSERT INTO d.scattered (key) SELECT * FROM generate_series(1, 1000);
          ALTER TABLE d.scattered SPLIT AT (SELECT * FROM generate_series(20, 980, 20));
          ALTER TABLE d.scattered SCATTER;
          ': dial tcp 127.0.0.1:33427: connect: connection refused
    panic.go:500: -- test log scope end --
--- FAIL: TestTenantStreamingMultipleNodes (426.49s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7575198?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7575198?buildTab=artifacts#/) on master @ [a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412](https://github.com/cockroachdb/cockroach/commits/a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412):

Fatal error:

```
panic: test timed out after 14m55s

```
Stack: 

```
goroutine 84415807 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2036 +0x8e
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x32
```
<details><summary>Log preceding fatal error</summary>
<p>

```
* github.com/cockroachdb/cockroach/pkg/util/admission.(*WorkQueue).startClosingEpochs.func1()
* 	github.com/cockroachdb/cockroach/pkg/util/admission/work_queue.go:419 +0x1dc
* created by github.com/cockroachdb/cockroach/pkg/util/admission.(*WorkQueue).startClosingEpochs
* 	github.com/cockroachdb/cockroach/pkg/util/admission/work_queue.go:392 +0x56
* 
* goroutine 51783196 [select, 2 minutes]:
* google.golang.org/grpc.newClientStreamWithParams.func4()
* 	google.golang.org/grpc/external/org_golang_google_grpc/stream.go:341 +0x92
* created by google.golang.org/grpc.newClientStreamWithParams
* 	google.golang.org/grpc/external/org_golang_google_grpc/stream.go:340 +0xb7e
* 
* goroutine 41355735 [select]:
* github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed.(*Processor).run(0xc013d7b080, {0x63276d0?, 0xc00b5bca80?}, 0x33?, 0xc018e5d368, 0xc00a6b42d0)
* 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed/processor.go:290 +0x4ff
* github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed.(*Processor).Start.func1({0x63276d0?, 0xc00b5bca80?})
* 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/rangefeed/processor.go:238 +0x3c
* github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
* 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:489 +0x146
* created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
* 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:480 +0x43b
* 
* goroutine 41231141 [sync.Cond.Wait]:
* sync.runtime_notifyListWait(0xc016de2a50, 0x49be)
* 	GOROOT/src/runtime/sema.go:517 +0x14c
* sync.(*Cond).Wait(0x477ebe0?)
* 	GOROOT/src/sync/cond.go:70 +0x8c
* github.com/cockroachdb/cockroach/pkg/kv/kvserver.(*raftScheduler).worker(0xc008defef0, {0x63276d0, 0xc021e6cc00})
* 	github.com/cockroachdb/cockroach/pkg/kv/kvserver/pkg/kv/kvserver/scheduler.go:293 +0x298
* github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx.func2()
* 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:489 +0x146
* created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunAsyncTaskEx
* 	github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:480 +0x43b
* 
* goroutine 41361438 [select, 4 minutes]:
* google.golang.org/grpc.newClientStreamWithParams.func4()
* 	google.golang.org/grpc/external/org_golang_google_grpc/stream.go:341 +0x92
* created by google.golang.org/grpc.newClientStreamWithParams
* 	google.golang.org/grpc/external/org_golang_google_grpc/stream.go:340 +0xb7e
* 
* goroutine 51779761 [select, 2 minutes]:
* github.com/cockroachdb/cockroach/pkg/util/tracing/grpcinterceptor.newTracingClientStream.func2()
* 	github.com/cockroachdb/cockroach/pkg/util/tracing/grpcinterceptor/grpc_interceptor.go:339 +0x71
* created by github.com/cockroachdb/cockroach/pkg/util/tracing/grpcinterceptor.newTracingClientStream
* 	github.com/cockroachdb/cockroach/pkg/util/tracing/grpcinterceptor/grpc_interceptor.go:338 +0x152
* 
* goroutine 44105313 [select, 4 minutes]:
* github.com/cockroachdb/cockroach/pkg/sql.(*runnerCoordinator).init.func1({0xc01936f000, 0xc001d317d0})
* 	github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:146 +0xf8
* github.com/cockroach
*
```
</p>
</details>
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7637337?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7637337?buildTab=artifacts#/) on master @ [dca415eddac0d659ae6d76b4e3dfdf4076adbd34](https://github.com/cockroachdb/cockroach/commits/dca415eddac0d659ae6d76b4e3dfdf4076adbd34):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3212535176
    test_log_scope.go:79: use -show-logs to present logs inline
panic: use of Span after Finish. Span: stream-ingestion-processor. Finish previously called at: <stack not captured. Set debugUseAfterFinish>

goroutine 57454762 [running]:
github.com/cockroachdb/cockroach/pkg/util/tracing.(*Span).detectUseAfterFinish(0xc00eb8bc58?)
	github.com/cockroachdb/cockroach/pkg/util/tracing/span.go:181 +0x12b
github.com/cockroachdb/cockroach/pkg/util/tracing.(*Tracer).startSpanGeneric(0xc0014520e0, {0x63341a8, 0xc00e4cd340}, {0x4f5a401, 0x16}, {{0x0}, 0x0, {0x0, 0x0, {{0x0, ...}, ...}, ...}, ...})
	github.com/cockroachdb/cockroach/pkg/util/tracing/tracer.go:1212 +0xa4e
github.com/cockroachdb/cockroach/pkg/util/tracing.(*Tracer).StartSpanCtx(0xc0051dd000?, {0x63341a8?, 0xc00e4cd340?}, {0x4f5a401?, 0x16?}, {0xc000ddec78?, 0x1?, 0x100000001?})
	github.com/cockroachdb/cockroach/pkg/util/tracing/tracer.go:1068 +0xe7
github.com/cockroachdb/cockroach/pkg/util/tracing.ChildSpan({0x63341a8, 0xc00e4cd340}, {0x4f5a401, 0x16})
	github.com/cockroachdb/cockroach/pkg/util/tracing/tracer.go:1562 +0x9c
github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamclient.(*partitionedStreamSubscription).Subscribe(0xc0131abc70, {0x63341a8?, 0xc00e4cd340?})
	github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamclient/partitioned_stream_client.go:283 +0x65
github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1()
	github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:168 +0x25
golang.org/x/sync/errgroup.(*Group).Go.func1()
	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:75 +0x64
created by golang.org/x/sync/errgroup.(*Group).Go
	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:72 +0xa5
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7910651?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7910651?buildTab=artifacts#/) on master @ [f2b00e8039af6ea8887ec124dad8daf19da6fbf1](https://github.com/cockroachdb/cockroach/commits/f2b00e8039af6ea8887ec124dad8daf19da6fbf1):

Fatal error:

```
panic: use of Span after Finish. Span: stream-ingestion-processor. Finish previously called at: <stack not captured. Set debugUseAfterFinish>

```
Stack: 

```
goroutine 72344009 [running]:
github.com/cockroachdb/cockroach/pkg/util/tracing.(*Span).detectUseAfterFinish(0xc00352a858?)
	github.com/cockroachdb/cockroach/pkg/util/tracing/span.go:181 +0x12b
github.com/cockroachdb/cockroach/pkg/util/tracing.(*Tracer).startSpanGeneric(0xc018dd50a0, {0x6aa7ef0, 0xc01455d780}, {0x55c3909, 0x16}, {{0x0}, 0x0, {0x0, 0x0, {{0x0, ...}, ...}, ...}, ...})
	github.com/cockroachdb/cockroach/pkg/util/tracing/tracer.go:1212 +0xa4e
github.com/cockroachdb/cockroach/pkg/util/tracing.(*Tracer).StartSpanCtx(0xc005fe3000?, {0x6aa7ef0?, 0xc01455d780?}, {0x55c3909?, 0x16?}, {0xc004f60c78?, 0x1?, 0x478ab0?})
	github.com/cockroachdb/cockroach/pkg/util/tracing/tracer.go:1068 +0xe7
github.com/cockroachdb/cockroach/pkg/util/tracing.ChildSpan({0x6aa7ef0, 0xc01455d780}, {0x55c3909, 0x16})
	github.com/cockroachdb/cockroach/pkg/util/tracing/tracer.go:1562 +0x9c
github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamclient.(*partitionedStreamSubscription).Subscribe(0xc025f5cd70, {0x6aa7ef0?, 0xc01455d780?})
	github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamclient/partitioned_stream_client.go:293 +0x65
github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1()
	github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:168 +0x25
golang.org/x/sync/errgroup.(*Group).Go.func1()
	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:75 +0x64
created by golang.org/x/sync/errgroup.(*Group).Go
	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:72 +0xa5
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3069824365
    test_log_scope.go:79: use -show-logs to present logs inline
```
</p>
</details>
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7954145?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/7954145?buildTab=artifacts#/) on master @ [052acc88ad9d7296ce6b8b441627fb469cc74d95](https://github.com/cockroachdb/cockroach/commits/052acc88ad9d7296ce6b8b441627fb469cc74d95):

Fatal error:

```
panic: use of Span after Finish. Span: stream-ingestion-processor. Finish previously called at: <stack not captured. Set debugUseAfterFinish>

```
Stack: 

```
goroutine 75731267 [running]:
github.com/cockroachdb/cockroach/pkg/util/tracing.(*Span).detectUseAfterFinish(0xc00d463c58?)
	github.com/cockroachdb/cockroach/pkg/util/tracing/span.go:181 +0x12b
github.com/cockroachdb/cockroach/pkg/util/tracing.(*Tracer).startSpanGeneric(0xc01dcd41c0, {0x6ad10d0, 0xc015500500}, {0x55e4a13, 0x16}, {{0x0}, 0x0, {0x0, 0x0, {{0x0, ...}, ...}, ...}, ...})
	github.com/cockroachdb/cockroach/pkg/util/tracing/tracer.go:1219 +0xa4e
github.com/cockroachdb/cockroach/pkg/util/tracing.(*Tracer).StartSpanCtx(0xc00d463000?, {0x6ad10d0?, 0xc015500500?}, {0x55e4a13?, 0x16?}, {0xc00b7a2c78?, 0x1?, 0x478ab0?})
	github.com/cockroachdb/cockroach/pkg/util/tracing/tracer.go:1075 +0xe7
github.com/cockroachdb/cockroach/pkg/util/tracing.ChildSpan({0x6ad10d0, 0xc015500500}, {0x55e4a13, 0x16})
	github.com/cockroachdb/cockroach/pkg/util/tracing/tracer.go:1572 +0x9c
github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamclient.(*partitionedStreamSubscription).Subscribe(0xc035ba4b40, {0x6ad10d0?, 0xc015500500?})
	github.com/cockroachdb/cockroach/pkg/ccl/streamingccl/streamclient/partitioned_stream_client.go:301 +0x65
github.com/cockroachdb/cockroach/pkg/util/ctxgroup.Group.GoCtx.func1()
	github.com/cockroachdb/cockroach/pkg/util/ctxgroup/ctxgroup.go:168 +0x25
golang.org/x/sync/errgroup.(*Group).Go.func1()
	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:75 +0x64
created by golang.org/x/sync/errgroup.(*Group).Go
	golang.org/x/sync/errgroup/external/org_golang_x_sync/errgroup/errgroup.go:72 +0xa5
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2697974159
    test_log_scope.go:79: use -show-logs to present logs inline
```
</p>
</details>
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8382291?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8382291?buildTab=artifacts#/) on master @ [d698e5eaf36bee19901beeff7db3425315402907](https://github.com/cockroachdb/cockroach/commits/d698e5eaf36bee19901beeff7db3425315402907):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes689947675
    test_log_scope.go:79: use -show-logs to present logs inline
    replication_stream_e2e_test.go:733: 
        	Error Trace:	/home/roach/.cache/bazel/_bazel_roach/c5a4e7d36696d9cd970af2045211a7df/sandbox/processwrapper-sandbox/2718/execroot/com_github_cockroachdb_cockroach/bazel-out/k8-fastbuild/bin/pkg/ccl/streamingccl/streamingest/streamingest_test_/streamingest_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go:733
        	Error:      	"1" is not greater than "1"
        	Test:       	TestTenantStreamingMultipleNodes
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes689947675
--- FAIL: TestTenantStreamingMultipleNodes (44.31s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8764658?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8764658?buildTab=artifacts#/) on master @ [e9c96e7179e19aae2f8d386f67eb950db8c3354b](https://github.com/cockroachdb/cockroach/commits/e9c96e7179e19aae2f8d386f67eb950db8c3354b):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1508340446
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:160: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1508340446
--- FAIL: TestTenantStreamingMultipleNodes (437.00s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8965708?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8965708?buildTab=artifacts#/) on master @ [1b162d1b274eec7b307fbbfca7294460bfdef025](https://github.com/cockroachdb/cockroach/commits/1b162d1b274eec7b307fbbfca7294460bfdef025):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2942092632
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:160: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2942092632
--- FAIL: TestTenantStreamingMultipleNodes (372.33s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8998198?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/8998198?buildTab=artifacts#/) on master @ [01e14f68f7c757cab1c878cebfecd73c5119046b](https://github.com/cockroachdb/cockroach/commits/01e14f68f7c757cab1c878cebfecd73c5119046b):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3930397034
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:160: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3930397034
--- FAIL: TestTenantStreamingMultipleNodes (344.00s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9012397?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9012397?buildTab=artifacts#/) on master @ [5b2a5670cbbe895d76602c230390816e783e0caa](https://github.com/cockroachdb/cockroach/commits/5b2a5670cbbe895d76602c230390816e783e0caa):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes98171905
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:160: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes98171905
--- FAIL: TestTenantStreamingMultipleNodes (356.95s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9020310?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9020310?buildTab=artifacts#/) on master @ [95684388bfbb372ac8c7e1a8ebd4a0d6f447e147](https://github.com/cockroachdb/cockroach/commits/95684388bfbb372ac8c7e1a8ebd4a0d6f447e147):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1421172729
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:160: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1421172729
--- FAIL: TestTenantStreamingMultipleNodes (361.83s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9051303?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9051303?buildTab=artifacts#/) on master @ [43f7cde6f036f720388912bde7310095c2f8ca9e](https://github.com/cockroachdb/cockroach/commits/43f7cde6f036f720388912bde7310095c2f8ca9e):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes4156616668
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:160: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes4156616668
--- FAIL: TestTenantStreamingMultipleNodes (364.32s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9071278?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9071278?buildTab=artifacts#/) on master @ [9b43dc50d468171bc9919dc684b7274b63e13da2](https://github.com/cockroachdb/cockroach/commits/9b43dc50d468171bc9919dc684b7274b63e13da2):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes433979448
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:160: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes433979448
--- FAIL: TestTenantStreamingMultipleNodes (388.89s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9088959?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9088959?buildTab=artifacts#/) on master @ [d68f5cdfb25476aaf874bff0bb6515eba02efcd2](https://github.com/cockroachdb/cockroach/commits/d68f5cdfb25476aaf874bff0bb6515eba02efcd2):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3642676411
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:160: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3642676411
--- FAIL: TestTenantStreamingMultipleNodes (381.90s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9107274?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9107274?buildTab=artifacts#/) on master @ [98594dda15e35643e447877ee591770083bb854c](https://github.com/cockroachdb/cockroach/commits/98594dda15e35643e447877ee591770083bb854c):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3281910321
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:160: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3281910321
--- FAIL: TestTenantStreamingMultipleNodes (396.51s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9122251?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9122251?buildTab=artifacts#/) on master @ [0a72a49eb6e4ce5823b53d02125441c3da2e94f5](https://github.com/cockroachdb/cockroach/commits/0a72a49eb6e4ce5823b53d02125441c3da2e94f5):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes895916156
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:160: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes895916156
--- FAIL: TestTenantStreamingMultipleNodes (372.70s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9131790?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9131790?buildTab=artifacts#/) on master @ [53dbb86acb1d48309530181b94838faf937084d3](https://github.com/cockroachdb/cockroach/commits/53dbb86acb1d48309530181b94838faf937084d3):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2071181513
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:160: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2071181513
--- FAIL: TestTenantStreamingMultipleNodes (386.32s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9142238?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9142238?buildTab=artifacts#/) on master @ [9acc75317aebfdfe69fb097a8a28e0040c7a67fd](https://github.com/cockroachdb/cockroach/commits/9acc75317aebfdfe69fb097a8a28e0040c7a67fd):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2002026208
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:160: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2002026208
--- FAIL: TestTenantStreamingMultipleNodes (376.74s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9162849?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9162849?buildTab=artifacts#/) on master @ [b89fa2cc4bc1fb9447ab1009b34b6f354f9618f0](https://github.com/cockroachdb/cockroach/commits/b89fa2cc4bc1fb9447ab1009b34b6f354f9618f0):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2958094229
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:160: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2958094229
--- FAIL: TestTenantStreamingMultipleNodes (414.41s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9183335?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9183335?buildTab=artifacts#/) on master @ [78bb7696847e1d8abf5d6a199c6dd3414a568691](https://github.com/cockroachdb/cockroach/commits/78bb7696847e1d8abf5d6a199c6dd3414a568691):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3369590575
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:160: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3369590575
--- FAIL: TestTenantStreamingMultipleNodes (401.73s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9184031?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9184031?buildTab=artifacts#/) on release-23.1 @ [80c4895c566a7eaa6f16c4098980509dd3795ad7](https://github.com/cockroachdb/cockroach/commits/80c4895c566a7eaa6f16c4098980509dd3795ad7):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3240993902
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:160: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3240993902
--- FAIL: TestTenantStreamingMultipleNodes (434.96s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9245088?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9245088?buildTab=artifacts#/) on master @ [0fcc33bc2870961b9387999d8a9fed97fccbb2ae](https://github.com/cockroachdb/cockroach/commits/0fcc33bc2870961b9387999d8a9fed97fccbb2ae):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes4103304204
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes4103304204
--- FAIL: TestTenantStreamingMultipleNodes (281.78s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9245299?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9245299?buildTab=artifacts#/) on release-23.1 @ [f351747ed97862fc037717cadec23f18073fb6be](https://github.com/cockroachdb/cockroach/commits/f351747ed97862fc037717cadec23f18073fb6be):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes101012290
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes101012290
--- FAIL: TestTenantStreamingMultipleNodes (290.11s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9264655?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9264655?buildTab=artifacts#/) on release-23.1 @ [186218edb86b5974501b2976cfd0704d4d6d1beb](https://github.com/cockroachdb/cockroach/commits/186218edb86b5974501b2976cfd0704d4d6d1beb):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes138162187
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes138162187
--- FAIL: TestTenantStreamingMultipleNodes (309.93s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9283734?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9283734?buildTab=artifacts#/) on master @ [143b63a6a27aeb286637dd2e5abddafdf0c51874](https://github.com/cockroachdb/cockroach/commits/143b63a6a27aeb286637dd2e5abddafdf0c51874):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1760748692
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1760748692
--- FAIL: TestTenantStreamingMultipleNodes (339.18s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9284119?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9284119?buildTab=artifacts#/) on release-23.1 @ [3551dc089ac3efcb3772df2109164dcb01e79d8f](https://github.com/cockroachdb/cockroach/commits/3551dc089ac3efcb3772df2109164dcb01e79d8f):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1783350766
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1783350766
--- FAIL: TestTenantStreamingMultipleNodes (305.94s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9307254?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9307254?buildTab=artifacts#/) on master @ [bb5c97b5d2073a27e7ec88654b05ac913112f610](https://github.com/cockroachdb/cockroach/commits/bb5c97b5d2073a27e7ec88654b05ac913112f610):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2205030705
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes2205030705
--- FAIL: TestTenantStreamingMultipleNodes (324.93s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9354450?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9354450?buildTab=artifacts#/) on master @ [4d23e874fb837311177aad60512aab334d6e64ea](https://github.com/cockroachdb/cockroach/commits/4d23e874fb837311177aad60512aab334d6e64ea):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1517214373
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1517214373
--- FAIL: TestTenantStreamingMultipleNodes (332.04s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9373832?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9373832?buildTab=artifacts#/) on master @ [99102ddf4b7602788b422366f1acc14b81c64d24](https://github.com/cockroachdb/cockroach/commits/99102ddf4b7602788b422366f1acc14b81c64d24):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes920076689
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes920076689
--- FAIL: TestTenantStreamingMultipleNodes (339.01s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9383697?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9383697?buildTab=artifacts#/) on master @ [1e27c65fb5a3810a5e2285cf07b45d95d401cca6](https://github.com/cockroachdb/cockroach/commits/1e27c65fb5a3810a5e2285cf07b45d95d401cca6):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1884243888
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1884243888
--- FAIL: TestTenantStreamingMultipleNodes (356.79s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9418170?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9418170?buildTab=artifacts#/) on master @ [07c7d4b3abfa1dc79d8fda385132a27c81665fee](https://github.com/cockroachdb/cockroach/commits/07c7d4b3abfa1dc79d8fda385132a27c81665fee):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1399596882
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1399596882
--- FAIL: TestTenantStreamingMultipleNodes (398.98s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9442840?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9442840?buildTab=artifacts#/) on master @ [ff2c4a3454a63bd74fe9a2feed53a7bdcfaf5191](https://github.com/cockroachdb/cockroach/commits/ff2c4a3454a63bd74fe9a2feed53a7bdcfaf5191):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes4156196780
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes4156196780
--- FAIL: TestTenantStreamingMultipleNodes (355.42s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9462549?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9462549?buildTab=artifacts#/) on master @ [530100fd39cc722bc324bfb3869a325622258fb3](https://github.com/cockroachdb/cockroach/commits/530100fd39cc722bc324bfb3869a325622258fb3):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1328195333
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1328195333
--- FAIL: TestTenantStreamingMultipleNodes (340.67s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9482850?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9482850?buildTab=artifacts#/) on master @ [8e6f530644eb35313a479af529b1f672c12dca38](https://github.com/cockroachdb/cockroach/commits/8e6f530644eb35313a479af529b1f672c12dca38):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes844536118
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes844536118
--- FAIL: TestTenantStreamingMultipleNodes (375.92s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9507971?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9507971?buildTab=artifacts#/) on master @ [85e41ca8d3d9edacf5ee3061a2591b159b9b0502](https://github.com/cockroachdb/cockroach/commits/85e41ca8d3d9edacf5ee3061a2591b159b9b0502):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3609601091
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3609601091
--- FAIL: TestTenantStreamingMultipleNodes (365.50s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9516909?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9516909?buildTab=artifacts#/) on master @ [85e41ca8d3d9edacf5ee3061a2591b159b9b0502](https://github.com/cockroachdb/cockroach/commits/85e41ca8d3d9edacf5ee3061a2591b159b9b0502):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3828767804
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3828767804
--- FAIL: TestTenantStreamingMultipleNodes (378.22s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9554759?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9554759?buildTab=artifacts#/) on master @ [582880ae2bb5d08e83a9871735deeeb3ed892e7d](https://github.com/cockroachdb/cockroach/commits/582880ae2bb5d08e83a9871735deeeb3ed892e7d):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes41291713
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes41291713
--- FAIL: TestTenantStreamingMultipleNodes (361.13s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9578697?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9578697?buildTab=artifacts#/) on master @ [678cfd4cbebbf3cced16747baf42a4c54cb2a92d](https://github.com/cockroachdb/cockroach/commits/678cfd4cbebbf3cced16747baf42a4c54cb2a92d):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1295164348
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1295164348
--- FAIL: TestTenantStreamingMultipleNodes (343.17s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9617078?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9617078?buildTab=artifacts#/) on master @ [51fcf796c5595549e90fca3cc1fd8165ee14fd6c](https://github.com/cockroachdb/cockroach/commits/51fcf796c5595549e90fca3cc1fd8165ee14fd6c):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes4068736999
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes4068736999
--- FAIL: TestTenantStreamingMultipleNodes (348.03s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9628376?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9628376?buildTab=artifacts#/) on master @ [c08946af1f935d543a033de6b7b585c4d65288c4](https://github.com/cockroachdb/cockroach/commits/c08946af1f935d543a033de6b7b585c4d65288c4):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes859396260
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes859396260
--- FAIL: TestTenantStreamingMultipleNodes (356.20s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9639373?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9639373?buildTab=artifacts#/) on master @ [3382b2737ceb72de9b14d1d5923773718be5bca7](https://github.com/cockroachdb/cockroach/commits/3382b2737ceb72de9b14d1d5923773718be5bca7):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3528973938
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3528973938
--- FAIL: TestTenantStreamingMultipleNodes (347.34s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9700615?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9700615?buildTab=artifacts#/) on master @ [b767225a6893ea6e81d50618d634dbba92d3b680](https://github.com/cockroachdb/cockroach/commits/b767225a6893ea6e81d50618d634dbba92d3b680):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1832890944
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1832890944
--- FAIL: TestTenantStreamingMultipleNodes (343.78s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9721890?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9721890?buildTab=artifacts#/) on master @ [8124bff03d073f35f4d2b6a2048c7f4417d757d9](https://github.com/cockroachdb/cockroach/commits/8124bff03d073f35f4d2b6a2048c7f4417d757d9):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1036103943
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1036103943
--- FAIL: TestTenantStreamingMultipleNodes (356.42s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9751547?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9751547?buildTab=artifacts#/) on master @ [23ea65e4b5cfaa5e02298cddfbe8f9fbe581456b](https://github.com/cockroachdb/cockroach/commits/23ea65e4b5cfaa5e02298cddfbe8f9fbe581456b):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1032645514
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1032645514
--- FAIL: TestTenantStreamingMultipleNodes (363.35s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9761682?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9761682?buildTab=artifacts#/) on master @ [1f3419e178bdba544f74d9c9e14a4682efd18028](https://github.com/cockroachdb/cockroach/commits/1f3419e178bdba544f74d9c9e14a4682efd18028):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes175360678
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes175360678
--- FAIL: TestTenantStreamingMultipleNodes (350.53s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9783062?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9783062?buildTab=artifacts#/) on master @ [847de4595b62a0c03b14a3d0865cc8222fcf0f61](https://github.com/cockroachdb/cockroach/commits/847de4595b62a0c03b14a3d0865cc8222fcf0f61):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1734790805
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1734790805
--- FAIL: TestTenantStreamingMultipleNodes (407.15s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9804943?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9804943?buildTab=artifacts#/) on master @ [2f4231084e27760b4927e15da9b6d46705dc8552](https://github.com/cockroachdb/cockroach/commits/2f4231084e27760b4927e15da9b6d46705dc8552):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1729029447
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:181: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes1729029447
--- FAIL: TestTenantStreamingMultipleNodes (295.89s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

ccl/streamingccl/streamingest.TestTenantStreamingMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9849978?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9849978?buildTab=artifacts#/) on master @ [3b1bc872bb4bf8e5eb2ad8790ba8ec781167dd1f](https://github.com/cockroachdb/cockroach/commits/3b1bc872bb4bf8e5eb2ad8790ba8ec781167dd1f):


```
=== RUN   TestTenantStreamingMultipleNodes
    test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3017851885
    test_log_scope.go:79: use -show-logs to present logs inline
    testutils.go:182: condition failed to evaluate within 2m0s: expected job status succeeded, but got running
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/90c1b75d835f45b8488807abb5b1092d/logTestTenantStreamingMultipleNodes3017851885
--- FAIL: TestTenantStreamingMultipleNodes (310.18s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101281 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1.0]
- #99900 ccl/streamingccl/streamingest: TestTenantStreamingMultipleNodes failed [C-test-failure O-robot T-disaster-recovery branch-release-23.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantStreamingMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/streamingccl/replicationtestutils/BUILD.bazel
+++ b/pkg/ccl/streamingccl/replicationtestutils/BUILD.bazel
@@ -39,6 +39,7 @@ go_library(
         "//pkg/testutils/sqlutils",
         "//pkg/testutils/testcluster",
         "//pkg/util/contextutil",
+        "//pkg/util/ctxgroup",
         "//pkg/util/hlc",
         "//pkg/util/protoutil",
         "@com_github_cockroachdb_apd_v3//:apd",

--- a/pkg/ccl/streamingccl/replicationtestutils/testutils.go
+++ b/pkg/ccl/streamingccl/replicationtestutils/testutils.go
@@ -35,6 +35,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/sqlutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/testcluster"
+	"github.com/cockroachdb/cockroach/pkg/util/ctxgroup"
 	"github.com/cockroachdb/cockroach/pkg/util/hlc"
 	"github.com/cockroachdb/cockroach/pkg/util/protoutil"
 	"github.com/cockroachdb/errors"
@@ -258,8 +259,26 @@ func CreateTenantStreamingClusters(
 		}
 	}
 
-	// Start the source cluster.
-	srcCluster, srcURL, srcCleanup := startTestCluster(ctx, t, serverArgs, args.SrcNumNodes)
+	g := ctxgroup.WithContext(ctx)
+
+	var srcCluster *testcluster.TestCluster
+	var srcURL url.URL
+	var srcCleanup func()
+	g.GoCtx(func(ctx context.Context) error {
+		// Start the source cluster.
+		srcCluster, srcURL, srcCleanup = startTestCluster(ctx, t, serverArgs, args.SrcNumNodes)
+		return nil
+	})
+
+	var destCluster *testcluster.TestCluster
+	var destCleanup func()
+	g.GoCtx(func(ctx context.Context) error {
+		// Start the destination cluster.
+		destCluster, _, destCleanup = startTestCluster(ctx, t, serverArgs, args.DestNumNodes)
+		return nil
+	})
+
+	require.NoError(t, g.Wait())
 
 	clusterSettings := cluster.MakeTestingClusterSettings()
 	for _, setting := range []*settings.BoolSetting{
@@ -275,9 +294,6 @@ func CreateTenantStreamingClusters(
 	}
 	srcTenantServer, srcTenantConn := serverutils.StartTenant(t, srcCluster.Server(0), tenantArgs)
 	waitForTenantPodsActive(t, srcTenantServer, 1)
-
-	// Start the destination cluster.
-	destCluster, _, destCleanup := startTestCluster(ctx, t, serverArgs, args.DestNumNodes)
 
 	tsc := &TenantStreamingClusters{
 		T:               t,
@@ -355,13 +371,12 @@ func CreateScatteredTable(t *testing.T, c *TenantStreamingClusters, numNodes int
 	// Create a source table with multiple ranges spread across multiple nodes
 	numRanges := 50
 	rowsPerRange := 20
-	c.SrcTenantSQL.Exec(t, fmt.Sprintf(`
-  CREATE TABLE d.scattered (key INT PRIMARY KEY);
-  INSERT INTO d.scattered (key) SELECT * FROM generate_series(1, %d);
-  ALTER TABLE d.scattered SPLIT AT (SELECT * FROM generate_series(%d, %d, %d));
-  ALTER TABLE d.scattered SCATTER;
-  `, numRanges*rowsPerRange, rowsPerRange, (numRanges-1)*rowsPerRange, rowsPerRange))
-	c.SrcSysSQL.CheckQueryResultsRetry(t, "SELECT count(distinct lease_holder) from crdb_internal.ranges", [][]string{{fmt.Sprint(numNodes)}})
+	c.SrcTenantSQL.Exec(t, "CREATE TABLE d.scattered (key INT PRIMARY KEY)")
+	c.SrcTenantSQL.Exec(t, "INSERT INTO d.scattered (key) SELECT * FROM generate_series(1, $1)",
+		numRanges*rowsPerRange)
+	c.SrcTenantSQL.Exec(t, "ALTER TABLE d.scattered SPLIT AT (SELECT * FROM generate_series($1::INT, $2::INT, $3::INT))",
+		rowsPerRange, (numRanges-1)*rowsPerRange, rowsPerRange)
+	c.SrcTenantSQL.Exec(t, "ALTER TABLE d.scattered SCATTER")
 }
 
 var defaultSrcClusterSetting = map[string]string{

--- a/pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go
+++ b/pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go
@@ -433,13 +433,12 @@ func TestTenantStreamingUnavailableStreamAddress(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
-	skip.WithIssue(t, 94738, "flaky test")
 	skip.UnderRace(t, "takes too long with multiple nodes")
 
 	ctx := context.Background()
 	args := replicationtestutils.DefaultTenantStreamingClustersArgs
 
-	args.SrcNumNodes = 3
+	args.SrcNumNodes = 4
 	args.DestNumNodes = 3
 
 	c, cleanup := replicationtestutils.CreateTenantStreamingClusters(ctx, t, args)
@@ -625,13 +624,12 @@ func TestTenantStreamingDeleteRange(t *testing.T) {
 func TestTenantStreamingMultipleNodes(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
-	skip.WithIssue(t, 86206)
 
 	skip.UnderRace(t, "takes too long with multiple nodes")
 
 	ctx := context.Background()
 	args := replicationtestutils.DefaultTenantStreamingClustersArgs
-	args.SrcNumNodes = 3
+	args.SrcNumNodes = 4
 	args.DestNumNodes = 3
 
 	// Track the number of unique addresses that were connected to
@@ -648,6 +646,8 @@ func TestTenantStreamingMultipleNodes(t *testing.T) {
 	c, cleanup := replicationtestutils.CreateTenantStreamingClusters(ctx, t, args)
 	defer cleanup()
 
+	// Make sure we have data on all nodes, so that we will have multiple
+	// connections and client addresses (and actually test multi-node).
 	replicationtestutils.CreateScatteredTable(t, c, 3)
 
 	producerJobID, ingestionJobID := c.StartStreamReplication(ctx)

